clojure_liberator/src/clojure_liberator/core.clj

```diff
@@ -1,20 +1,43 @@
 (ns clojure-liberator.core
-  (:gen-class)
+  ;(:gen-class)
   (:require [liberator.core :refer [resource defresource]]
             [ring.middleware.params :refer [wrap-params]]
             [compojure.core :refer [defroutes ANY]]
-            [ring.adapter.jetty :as jetty]))
+            [ring.adapter.jetty :as jetty]
+            [clojure-liberator.data-access :as data-access]
+            [clojure.data.json :as json]
+            [clj-time.format :as fmt]
+            [clj-time.core :as t]
+            clj-time.jdbc
+            )
+  (:import (org.joda.time DateTime)))
+
+(def formatter (fmt/formatters :date-time-no-ms))
+
+(extend-type DateTime
+  json/JSONWriter
+  (-write [date out]
+    (json/-write (fmt/unparse formatter date) out)))
```

Comment:
notes: `json/JSONWriter` is a clojure protocol. We are adding that protocol to the joda DateTime class, so that it can be serialized as json correctly.  The protocol function is `-write` and the implementation uses the `:date-time-no-ms` formatter from the `clj-time.format` namespace.